tools/preload.py

```diff
@@ -63,6 +63,8 @@ def get_absolute_url(domain, path, icon):
 
     if(path.startswith('http')):
```

Comment:
Is this another hack? how come has_scheme(icon) is false and path.stratswith('http') is true? Do they give `http/<some-path> ` a special meaning? I'm trying to assess the consequences on this patch...